spec/services/user_video_creator_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe UserVideoCreator do
+  it 'creates or initializes a video model associated with a user' do
+    youtube_id = 'abcdefg'
```

Comment:
This is kind of ugly - the next time I have to touch this I will probably refactor to simplify how I'm using test doubles.